File: tasks/task12_1/point.py
from math import hypot

class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y
    def distance(self):
        return hypot(self.x,self.y)
    def distanceTo(self, p):
        return hypot(self.x - p.x, self.y - p.y)

x = int(input("Input: "))
y = int(input("Input: "))
p = Point(x,y)

print(p.distanceTo(Point(0,0)))
print(p.distanceTo(Point(4.5,7.0)))

File: tasks/task12_1/Point.kt
import kotlin.math.hypot


class Point(var x: Double, var y: Double) {
    fun distance(): Double =
        hypot(x, y)
    fun distanceTo(p: Point): Double =
        hypot(x - p.x, y - p.y)
}

fun main(){
    val x = readln().toDouble()
    val y = readln().toDouble()
    val p = Point(x,y)

    println(p.distanceTo(Point(0.0,0.0)))
    println(p.distanceTo(Point(4.5,7.0)))
}